### PR TITLE
Add Screenshots of the app

### PR DESCRIPTION
**Pull request by roninJosue**
The README file has been updated.

Screenshots added and used APIs.

PR for [#1](https://github.com/dmahajan980/node-weather-app/issues/1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,15 @@ A weather forecasting app built on node.js (express.js).
    ```bash
    npm run test
    ```
+
+## Screen Shots
```
Comment (dmahajan980) [inline README.md]: "User Interface" sounds nicer.

File: README.md [hunk 2]
```diff
@@ -16,3 +16,15 @@ A weather forecasting app built on node.js (express.js).
...
+### Weather page
```
Comment (dmahajan980) [inline README.md]: You don't need to add screenshots of all the pages. You can simply add two "full-window screenshot" of the homepage (with and without search results). You can also use a single full-page GIF here, demonstrating how the app works. With those, you can get rid of the other screenshots accordingly.

**Review by dmahajan980 — CHANGES_REQUESTED** [review]

**Comment by dmahajan980** [comment]
Hi @roninJosue, apologies for the delay. I've suggested some changes. Kindly got through them and let me know if you have any doubts. Once you submit those changes, we can proceed with merging the pull :)

**Comment by roninJosue** [comment]
Hello @dmahajan980 I will make the suggested changes, I just have a question, is another PR made to upload those changes?

File: README.md (hunk 1 above)
Comment (roninJosue) [reply README.md]: Screen Shots change for User Interface

File: README.md (hunk 2 above)
Comment (roninJosue) [reply README.md]: Gif added showing how the app works

**Comment by roninJosue** [comment]
Hello @dmahajan980 i submit the changes

**Comment by dmahajan980** [comment]
Good job @roninJosue 😃 
Everything seems awesome. Thanks for all your patience and hard work ✨

**Comment by roninJosue** [comment]
Thank you very much 👍 for the opportunity you gave me to make the Pull Request. Greetings.

**Pull request merged**